main.py

```diff
@@ -0,0 +1,47 @@
+from flask import Blueprint, Flask, request, jsonify
+from models import CelestialBodies, Landmark, User, Passenger
+
+main  = Blueprint('main', __name__)
+@main.route('/')
+def hello():
+    return jsonify({'message': "App is running"})
```

Comment:
Creating this file and blueprint allows us to have different instances of the app such as in the tests get access to the routes and the associated actions.